src/shared/pages/list/mpamBusinessUnits/selectBusinessArea.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { RouteComponentProps } from 'react-router';
+import { Link } from 'react-router-dom';
+import { string, object } from 'yup';
+import Submit from '../../../common/components/forms/submit';
+import { ApplicationConsumer } from '../../../contexts/application';
+import ErrorSummary from '../../../common/components/errorSummary';
+import useError from '../../../hooks/useError';
+import * as constants from '../../../models/constants';
+import { validate } from '../../../validation';
+import TypeAhead from '../../../common/components/typeAhead';
+import Item from '../../../models/item';
+
+interface ChooseBusinessAreaProps extends RouteComponentProps {
+    csrfToken?: string;
+}
+
+const validationSchema = object({
+    value: string()
+        .required()
+        .label('Business Area')
+});
+
+const ChooseBusinessArea: React.FC<ChooseBusinessAreaProps> = ({ csrfToken, history }) => {
+
+    const [pageError, addFormError, clearErrors] = useError('', constants.VALIDATION_ERROR_TITLE);
+
+    const [selectedCaseType, setSelectedCaseType] = React.useState<Item>();
+
+    const items: Item[] = [
+        { label: 'UKVI', value: 'MPAM_BUS_UNITS_1' },
+        { label: 'BF', value: 'MPAM_BUS_UNITS_2' },
+        { label: 'IE', value: 'MPAM_BUS_UNITS_3' },
+        { label: 'EUSS', value: 'MPAM_BUS_UNITS_4' },
+        { label: 'HMPO', value: 'MPAM_BUS_UNITS_5' },
+        { label: 'Windrush', value: 'MPAM_BUS_UNITS_6' },
+        { label: 'Corona Virus', value: 'MPAM_BUS_UNITS_7' },
```

Comment:
Duplication of strings for the labels here from listNames.tsx, potential to use the constant values?

Reply:
swapped to generate this array from listNames.tsx